birdhouse/optional-components/all-public-access/config/geoserver/permissions.cfg

```diff
@@ -0,0 +1,79 @@
+permissions:
+ - service: geoserver
+   resource: /
+   type: route
+   permission: read
+   group: anonymous
+   action: create
+ - service: geoserver
+   permission: describestoredqueries
+   group: anonymous
+   action: create
+ - service: geoserver
+   permission: describelayer
+   group: anonymous
+   action: create
+ - service: geoserver
+   permission: dropstoredquery
+   group: anonymous
+   action: create
+ - service: geoserver
+   permission: lockfeature
+   group: anonymous
+   action: create
+ - service: geoserver
+   permission: getmap
+   group: anonymous
+   action: create
+ - service: geoserver
+   permission: execute
+   group: anonymous
+   action: create
```

Comment:
This is not relevant if `wps: false`.
I don't think it would break, so can be left there with a comment regarding `wps` option for reference.